tests/unit/TableTest.php

```diff
@@ -121,14 +102,7 @@ public function testDeleteRecord()
 
     public function testQueryRecordMultipleCriteria($value='')
```

Comment:
We'll use the opportunity, could you remove this snippet:
```php
$value = ''
```

Reply:
removed

Reply:
I don't see it removed on the changeset,  did you push your latest version?